dts/msm8974/msm8974-lge-hammerhead.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0x7e 0x96 0x20002 0x0b>;
+
+	hammerhead {
+		model = "LG Google Nexus 5";
+		compatible = "lge,hammerhead", "qcom,msm8974", "lk2nd,device";
+		lk2nd,match-bootloader = "HHZ20h";
```

Comment:
Not sure if the 1-to-1 match works here, I suppose `HH` means `hammerhead` and the reminder is the version code. Are you sure there is no other hammerhead that might have other bootloader string?

Reply:
It may vary when using older firmware/bootloader. Latest firmware is recommended before anything funny, though.
I have D820 and D821. Both of them are upgraded to latest stock firmware and they are ``HHZ20h``.

Reply:
Looked for some photos just now. The bootloader version/code is formatted in ``HHZ``, ``2 number digits`` and ``1 lower case character``. So am I changing ``lk2nd,match-bootloader = "HHZ*"``?

Reply:
Yeah, IMO we shouldn't assume that the user will always have the latest firmware